core/src/main/web/app/mainapp/components/notebook/cell-directive.js

```diff
@@ -217,6 +217,30 @@
         $scope.isCodeCell = function() {
           return $scope.cellmodel.type == 'code';
         };
+
+        $scope.isSection = function() {
```

Comment:
I don't think we need this function. You can just ng-if="cellmodel.type === 'section'"